### ISSUE TITLE
Question about the hyper-parameter of SSV2?

### ISSUE THREAD

**Issue by PeiqinZhuang** [facebookresearch/Motionformer#13]
Hi, may I ask the extract learning rate when setting the batch size as 64 in SSV2.  Currently, it is set as 1e-4 in the provided config, while it validates the scaling rule, e.g. 32/256 * 1e-4, which has been mentioned for K400.
